.github/workflows/auto_import.yaml

```diff
@@ -14,19 +14,23 @@ jobs:
       with:
         go-version: '^1.16.4'
     - run: |
-        wget http://pypa-advisory-db.storage.googleapis.com/triage/pypi_links.json
-        wget http://pypa-advisory-db.storage.googleapis.com/triage/pypi_versions.json
+        wget https://pypa-advisory-db.storage.googleapis.com/triage/pypi_links.json
+        wget https://pypa-advisory-db.storage.googleapis.com/triage/pypi_versions.json
     - run: |
-        wget https://storage.googleapis.com/cve-osv-conversion/nvd/nvdcve-2.0-2024.json
+        for year in {2000..2030}; do
+          wget https://storage.googleapis.com/cve-osv-conversion/nvd/nvdcve-2.0-$year.json;
```

Comment:
NB: `wget https://storage.googleapis.com/cve-osv-conversion/nvd/nvdcve-2.0-2030.json` will 404, which _might_ fail the whole step/job here, unless I'm missing something.

Reply:
(To work around this, we could probably cap to `date +%Y` instead.)

Reply:
Great idea, thanks! Done in https://github.com/pypa/advisory-database/pull/230/commits/bb5d1280a3c36b5cf9ecd55b2fd1119a3cafe18f